assets/clients/clients.yaml

```diff
@@ -4,98 +4,109 @@ clients:
     oss: https://github.com/jellyfin/jellyfin-media-player
     official: true
     downloads:
-      - icon: github
-        url: https://github.com/jellyfin/jellyfin-media-player/releases
-
-      - icon: flathub
-        url: https://flathub.org/apps/details/com.github.iwalton3.jellyfin-media-player
+      -
+        type: github
```

Comment:
Do we really want new lines after the bullet point?

Reply:
I think that this makes it easier to keep several downloads apart,

```yaml
- type: github
  owner: test
  repo: test
- type: demo
  url: https://example.com
```

vs.

```yaml
- type: github
  owner: test
  repo: test

- type: demo
  url: https://example.com
```

vs.

```yaml
- 
  type: github
  owner: test
  repo: test
- 
  type: demo
  url: https://example.com
```